armi/settings/caseSettings.py

```diff
@@ -272,9 +272,20 @@ def loadFromInputFile(self, fName, handleInvalids=True, setPath=True):
         reader, path = self._prepToRead(fName)
         reader.readFromFile(fName, handleInvalids)
         self._applyReadSettings(path if setPath else None)
+        self.registerUserPlugins()
 
         return reader
 
+    def registerUserPlugins(self):
+        """TODO"""
+        userPlugins = self.__settings.get("userPlugins", [])
+        userPlugins = self["userPlugins"]
```

Comment:
can you add a comment here describing what is going on. Its surprising to set `userPlugins` 2x but in different ways

Reply:
also there is still todo in doc string, but i imagine you planned to update that.

Reply:
> can you add a comment here describing what is going on. Its surprising to set `userPlugins` 2x but in different ways

Ah, that's a typo.